tools/flambda_backend_objinfo.ml

```diff
@@ -115,8 +115,15 @@ let print_cma_infos (lib : Cmo_format.library) =
   printf "\n";
   List.iter print_cmo_infos lib.lib_units
 
-let print_cmi_infos name crcs =
+let print_cmi_infos name crcs kind =
+  let open Cmi_format in
   printf "Unit name: %a\n" Compilation_unit.output name;
+  let is_param =
+    match kind with
+    | Normal -> false
+    | Parameter -> true
+  in
+  printf "Is parameter: %s\n" (if is_param then "YES" else "no");
```

Comment:
(Same comment as the similar code in `tools/objinfo.ml`)

Reply:
See previous. (There should really be a `print_yes_or_no` function somewhere in the file for this.)